### PR TITLE
Scastie UI fix issues redesign

### PR DESCRIPTION
**Pull request by dominv**
This PR contains changes related to fix some issues and add some new features in the design and responsive support of **Scastie**. 

Here is a summary of the changes:

- Increased font size.
- **Scastie's gitter** button's text now shows inline.
- Github user's name appears in topbar.
- Fixed forced desktop version in mobile screens.
- When a code snippet is opened now the apps goes to the Editor's view.
- Added meta data and image to share in social media.
- Fixed **Save** button's behaviour.
- Review **Clear** button's behaviour.
- Added **New** button to create a new code snippet. 

@heathermiller @MasseGuillaume could you please review it?

Thanks!

### CODE REVIEW

**Comment by heathermiller** [comment]
🎉

File: client/src/main/scala/com.olegych.scastie.client/App.scala [hunk 1]
```diff
@@ -56,7 +59,16 @@ object App {
       .componentWillMount { current =>
         current.backend.start(current.props) >> setTitle(current.state)
       }
-      .componentDidMount(_.backend.setDimensions())
+      .componentDidMount{ s =>
+
+        val detectGesture = Callback(
```
Comment (MasseGuillaume) [inline client/src/main/scala/com.olegych.scastie.client/App.scala]: The scalajs-react way to do this is with HtmlRef. Let me apply that change right after I merge. It's ok for now,

File: client/src/main/scala/com.olegych.scastie.client/AppBackend.scala [hunk 2]
```diff
@@ -113,16 +113,39 @@ class AppBackend(scope: BackendScope[AppProps, AppState]) {
     socket
   }
 
+  def newSnippet(e: ReactEventI): Callback = {
```
Comment (MasseGuillaume) [inline client/src/main/scala/com.olegych.scastie.client/AppBackend.scala]: :-)

File: client/src/main/scala/com.olegych.scastie.client/CodeSnippets.scala [hunk 3]
```diff
@@ -78,7 +89,7 @@ object CodeSnippets {
             div(`id` := "snippets")(
               summaries.groupBy(_.snippetId.base64UUID).map {
                 case (base64UUID, groupedSummaries) =>
-                  div(`class` := "group")(
+                  div(`class` := "group", `key` := base64UUID)(
```
Comment (MasseGuillaume) [inline client/src/main/scala/com.olegych.scastie.client/CodeSnippets.scala]: This is to fix a warning right ?

**Comment by MasseGuillaume** [comment]
Thanks!

**Comment by MasseGuillaume** [comment]
Redeploying now.

File: client/src/main/scala/com.olegych.scastie.client/CodeSnippets.scala (hunk 3 above)
Comment (dominv) [inline client/src/main/scala/com.olegych.scastie.client/CodeSnippets.scala]: That's right!

File: client/src/main/scala/com.olegych.scastie.client/App.scala (hunk 1 above)
Comment (dominv) [inline client/src/main/scala/com.olegych.scastie.client/App.scala]: Of course.  :+1:

Comment (MasseGuillaume) [inline client/src/main/scala/com.olegych.scastie.client/App.scala]: https://github.com/scalacenter/scastie/commit/06c79a775da361d5043012c903d18139116b7e46#diff-a2fe55369a2d7906f6b21ce5698e7ef9R13 Done ;-)

**Comment by MasseGuillaume** [comment]
Redeployed.

**Comment by MasseGuillaume** [comment]
@heathermiller make shure wou hit `Cmd + Shift + R`

I created #130 related to tagging.

**Comment by heathermiller** [comment]
Yes @MasseGuillaume, I refreshed my cache before raising an issue. It still doesn't work.

File: client/src/main/scala/com.olegych.scastie.client/App.scala (hunk 1 above)
Comment (dominv) [inline client/src/main/scala/com.olegych.scastie.client/App.scala]: Great! Thanks!

**Comment by dominv** [comment]
@heathermiller @MasseGuillaume I've just created a PR https://github.com/scalacenter/scastie/pull/166 that solves the fonts issue and I've modified the Console's colours for the **Light** theme.

**Pull request merged**